### ISSUE TITLE
Keep track of bundle size for extensions via a non-blocking check

### ISSUE THREAD

**Issue by rsimha** [ampproject/amp-github-apps#271]
See https://github.com/ampproject/amphtml/issues/20843

**Comment by rsimha** [comment]
@danielrozenberg could you move the work plan to a separate comment authored by you? Right now, I'm getting notified each time the comment is edited.

**Comment by danielrozenberg** [comment]
lol sure, I'll remove it from the description and add it here:

Workplan:
- [x] submit and deploy #428
- [x] after #428 is deployed, create a PR that migrates all `.br` files to `.json` (with the proper format of course) → done at https://github.com/ampproject/amphtml-build-artifacts/pull/18
- [x] unrollback #412
- [x] create a new PR that reads bundle-sizes from the `.json` file instead of the `.br` file, submit and deploy it → done at #514
- [x] create a PR to receive a JSON payload with multiple bundle sizes when storing → done at #515
- [x] once #514 and #515 are deployed, update the amphtml repo to send to the JSON payload on master builds → done at https://github.com/ampproject/amphtml/pull/24955
- [x] do a switcharoo from /store.json to /store (both on bundle-size bot and in amphtml, make sure to keep in sync) → done at #519 and https://github.com/ampproject/amphtml/pull/24958 and #523
- [ ] start storing extensions and other runtime files for master push builds  → done at https://github.com/ampproject/amphtml/pull/25042
- [x] create a PR to receive a JSON payload with multiple bundle sizes when reporting → done at #527
- [ ] once deployed, update the amphtml repo to report the JSON payload on pull requests → done at https://github.com/ampproject/amphtml/pull/25042
- [ ] do a switcharoo from /report.json to /report (wait at least 2 weeks before doing this, to reduce changes of errors in straggling PRs)
- [ ] delete all `.br` and extensionless files from https://github.com/ampproject/amphtml-build-artifacts/tree/master/bundle-size
- [ ] decide how to report and block PRs that increase bundle size of any files other than `dist/v0.js`

**Comment by danielrozenberg** [comment]
Closing since the "keep track" part is done. Work continues in #617
